Mu2eG4/geom/geom_run1_a.txt

```diff
@@ -100,7 +100,9 @@ bool   vd.solid          = false;
 #include "Offline/Mu2eG4/geom/visualization_regions.txt"
 
 
-
+// patch to change East DS hatch block height
+double building.dsArea.hatchblock.E.offsetFromFloorSurface.y = 6400.80;
+double building.dsArea.hatchblock.E.yHalfThickness           = 457.20;
```

Comment:
I remember the 3ft thickness (=457.2mm halfthickness), but I don't know the other value.

Reply:
@oksuzian Could you please check these values?

Reply:
Looks right to me.